crates/test-utils/src/fixtures.rs

```diff
@@ -5,16 +5,16 @@ use crate::execution::contract::KakarotEvmContract;
 use crate::sequencer::Katana;
 
 #[fixture]
-#[awt]
 pub async fn counter(#[future] katana: Katana) -> (Katana, KakarotEvmContract) {
+    let katana = katana.await;
     let eoa = katana.eoa();
     let contract = eoa.deploy_evm_contract("Counter", ()).await.expect("Failed to deploy Counter contract");
     (katana, contract)
 }
 
 #[fixture]
-#[awt]
```

Comment:
why did you remove this?

Reply:
https://github.com/kkrt-labs/kakarot-rpc/pull/656#discussion_r1400738718